File: app/src/main/java/com/xiao/embeddedcar/Entity/LoginInfo.java
package com.xiao.embeddedcar.Entity;

import androidx.annotation.NonNull;

public class LoginInfo {
    //设备IP
    private String IP;
    //摄像头IP
    private String IPCamera = null;
    private String pureCameraIP = null;

    public String getIP() {
        return IP;
    }

    public void setIP(String IP) {
        this.IP = IP;
    }

    public String getIPCamera() {
        return IPCamera;
    }

    public void setIPCamera(String IPCamera) {
        this.IPCamera = IPCamera;
    }

    public String getPureCameraIP() {
        return pureCameraIP;
    }

    public void setPureCameraIP(String pureCameraIP) {
        this.pureCameraIP = pureCameraIP;
    }

    @NonNull
    @Override
    public String toString() {
        return "LoginInfo{" +
                "IP='" + IP + '\'' +
                ", IPCamera='" + IPCamera + '\'' +
                ", pureCameraIP='" + pureCameraIP + '\'' +
                '}' + "\n";
    }
}

File: app/src/main/java/com/xiao/embeddedcar/data/Entity/LoginInfo.kt
package com.xiao.embeddedcar.data.Entity

data class LoginInfo(
    //设备IP
    var iP: String? = null,

    //摄像头IP
    var iPCamera: String? = null,

    @JvmField
    var pureCameraIP: String? = null
) {
    override fun toString(): String {
        return """
               LoginInfo{IP='${iP}', IPCamera='${iPCamera}', pureCameraIP='$pureCameraIP'}
               """.trimIndent()
    }
}